web/src/admin.js

```diff
@@ -218,7 +218,9 @@ export function build_page() {
     $("#settings_content .organization-box").html(rendered_admin_tab);
     $("#settings_content .alert").removeClass("show");
 
-    settings_bots.update_bot_settings_tip();
+    settings_bots.update_bot_settings_tip($("#admin-bot-settings-tip"), true);
+    settings_bots.update_bot_settings_tip($("#personal-bot-settings-tip"), false);
```

Comment:
Should this only need to call the one for the organization settings page? And similarly, shouldn't the other copy only need to call the one for the page it's rendering?

Doing both in both places feels like a weird abstraction violation.

Reply:
No, this functions needs to call for both settings page. Just to clarify this function is handling initial rendering of both the settings page and other function is handling live update of both pages.

Reply:
Yes, I missed this. We should call this function for organization settings from here and for personal settings we should call it from `settings.launch`, which is called to render the personal settings pages.

Reply:
Okk, I will update the PR accordingly.